galley/pkg/server/server.go

```diff
@@ -183,7 +186,10 @@ func newServer(a *Args, p patchTable) (*Server, error) {
 
 	s.mcp = server.New(options, checker)
 
-	serverOptions := &source.ServerOptions{AuthChecker: checker}
+	serverOptions := &source.ServerOptions{
+		AuthChecker: checker,
+		RateLimiter: rate.NewLimiter(rate.Every(time.Second), 100),
```

Comment:
Let's not add hard-wired values here. These should be configurable.

Reply:
If you don't want to plumb them in now, please open a github issue and add a TODO referencing it. That way we can track and fix it later.

Reply:
I will create a GH issue and assign it to myself todo them later, the reason being I'm really hoping to keep the scope of this PR as small as possible.